content/04.results.md

```diff
@@ -174,12 +175,38 @@ We performed geneset variant analysis (GSVA) for Hallmark cancer gene sets, demo
+The classifier did not perform well on the poly-A sample subset (**Figure {@fig:FigS5}A-B**), likely due to the low number of poly-A samples in our dataset (N = 58), as we previously reported high accuracy of this classifier on another poly-A dataset [@doi:10.1016/j.celrep.2019.09.071].
+We annotated _TP53_ alterations as "activated" if samples harbored one of p.R273C or p.R248W mutations [@doi:10.1038/ng0593-42], "loss" if the patient had a Li Fraumeni Syndrome (LFS) predisposition diagnosis or if any other SNV, CNV, SV, or fusion fell within a functional domain.
+If the _TP53_ mutation did not reside within the DNA-binding domain or if an alteration was not detected in _TP53_, we annotated the tumor as "other".
+Surprisingly, we observed that samples annotated as either "activated" or "loss" had significantly higher _TP53_ scores than those annotated as "other" (**Figure {@fig:Fig5}B**, p<sub>adj</sub>loss vs. other < 2e-16, p<sub>adj</sub> activated vs. other = 4.0e-5), suggesting that the classifier detects an oncogenic phenotype of _TP53_ (scores > 0.5) rather than solely _TP53_ inactivation.
+Moreover, tumors with "activating" _TP53_ mutations had evidence of higher _TP53_ expression than those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig5}C**.
+To further validate the classifier's accuracy, we plotted _TP53_ scores for patients with LFS **Figure {@fig:Fig5}C**, hypothesizing all of these tumors would have high scores.
+Indeed, we observed significantly higher scores (Wilcoxon p = 4.4e-2) in LFS tumors for which we detected high-confidence _TP53_ somatic alterations. 
+Of note, we were unable to detect canonical somatic _TP53_ mutations in two LFS patient tumors, annotated as "other" in **Figure {@fig:Fig5}D**.
+We confirmed the LFS diagnosis from pathology reports and found each to have a germline variant in _TP53_, however, the tumor purity of these samples was low (16% and 37%), suggesting the classifier might require a certain level of tumor purity for high accuracy.
+Additionally, we found a significant positive correlation between _TP53_ scores and CNV breakpoint density (**Figure {@fig:Fig5}E**), consistent with our previously report [@doi:10.1016/j.celrep.2019.09.071].
+Tumors with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations: choroid plexus tumors, embryonal tumors, HGATs, and pineal tumors (**Figure {@fig:Fig5}F**), while melanocytic tumors, meningiomas, and tumors of  cranial and paraspinal nerves having the lowest scores.
+
+We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
```

Comment:
"Next" snuck back in!
```suggestion
We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (EXTEND) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; 10.1093/carcin/bgp268].
```

Reply:
Ha! What can I say- creature of habit trying to tell a story in sequence!

Reply:
I am going to leave this for now so we can talk about this figure stepwise to tell a story.